allennlp/data/vocabulary.py

```diff
@@ -134,12 +144,26 @@ class Vocabulary:
         The default is ``("*tags", "*labels")``, so as long as your namespace ends in "tags" or
         "labels" (which is true by default for all tag and label fields in this code), you don't
         have to specify anything here.
+    pretrained_files : ``Dict[str, str]``, optional
+        If provides, this map specifies the path to optional pretrained embedding files for each
+        namespace. This can be used to either restrict the vocabulary to only words which appear
+        in this file, or to ensure that any words in this file are included in the vocabulary
+        regardless of their count, depending on the value of ``pretrained_exclusive``.
```

Comment:
It'd be nice if this were more clear about what happens to words that you _didn't_ see in your data, but are in the embedding file.  You currently (and I think correctly) leave those out of the vocab, but that's not clear from this note.